### ISSUE TITLE
Problem with Mysql compatibility 5.7

### ISSUE THREAD

**Issue by prhost** [copona/copona#179]
Expression #2 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'copona.cp_order.date_added' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by - SQL query: SELECT COUNT(*) AS total, date_added FROM `cp_order` WHERE order_status_id IN('5','3') AND DATE(date_added) >= '2017-09-1' GROUP BY DATE(date_added)

We have this problem when using mysql 5.7.
at first it is not a very serious problem, because of the disable in sql_mode:

`mysql > SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));`

**Comment by arnisjuraga** [comment]
I suppose, this is not the only place, where full group by should be added, isn't it? 
This is one of the first differences you notice, when comparing MySQL group selects with other SQL engines, which uses full group by default.  Didn't know, that 5.7 turns it on by default

**Comment by prhost** [comment]
it seems that it is already activated, we need to fix it in places that have problem, or say that Copona does not support this version of mysql. Shared servers that have this active may be a problem when changing your Mysql variables

**Comment by arnisjuraga** [comment]
I agree on changing it everywhere and move forward for supporting new MySQL version.

**Comment by lucasjkr** [comment]
`SELECT COUNT(o.order_id) AS total, **DATE(**date_added**)**
FROM cp_order o
WHERE o.order_status_id IN('5','3') AND DATE(o.date_added) >= '2017-09-1' 
GROUP BY DATE(date_added)`

Is this the fix? Not getting an error, but don't have dummy data at the moment to check that its getting the proper results.

**Comment by lucasjkr** [comment]
Tried to bold **DATE(**date_added**)**, didn't work!

**Comment by prhost** [comment]
@lucasjkr yes, adding the DATE function solved the problem, thanks
